src/input.cpp

```diff
@@ -26,14 +26,12 @@
 #include "options.h"
 #include "output.h"
 #include "path_info.h"
+#include "point.h"
 #include "popup.h"
 #include "string_formatter.h"
 #include "string_input_popup.h"
 #include "translations.h"
 #include "ui_manager.h"
```

Comment:
It is required for code guarded by TILES define (`get_window_dimensions`, etc). Maybe we should guard this include too or move called function to a different header?

```suggestion
#include "ui_manager.h"
#include "sdltiles.h"
```